tests/test_naive_bayes.py

```diff
@@ -810,4 +810,89 @@ def test_io_from_samples():
 	logp1 = nb1.log_probability(X)
 	logp2 = nb2.log_probability(X)
 
-	assert_array_almost_equal(logp1, logp2)
\ No newline at end of file
+	assert_array_almost_equal(logp1, logp2)
+
+def test_propagation_pseudocount():
+	"""
+	Test that naive Bayes pseudocounts are passed to BernoulliDistribution.
+	"""
+	X_train = np.array([
+		[0., 1.],
+		[0., 1.],
+		[0., 1.],
+	])
+	y_train = np.array([0, 0, 1])
+
+	# Test that both `alpha` and `pseudocount` are propagated to the
+	# BernoulliDistribution.
+	for kwargs in ({'alpha': 1.0}, {'pseudocount': 1.0}):
+		pmodel = NaiveBayes.from_samples(
+			BernoulliDistribution,
+			X_train,
+			y_train,
+			**kwargs
+		)
+		# Check fit for y=0 label (2 records + pseudocount=1 per category).
```

Comment:
This test will probably need to be changed. We shouldn't have two parameters that do the same thing. pseudocount should be on the labels (and so only affect the NaiveBayes priors) and alpha should be on the counts for the underlying distributions. That way pseudocount works the exact same way for normal distributions as bernoulli.